src/hermes/commands/init/base.py

```diff
@@ -0,0 +1,244 @@
+# SPDX-FileCopyrightText: 2022 German Aerospace Center (DLR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileContributor: Michael Meinel
+
+import argparse
+import os
+import subprocess
+
+import requests
+from pydantic import BaseModel
+
+from hermes.commands.base import HermesCommand
+import hermes.commands.init.oauth_github as oauth_github
+import hermes.commands.init.oauth_zenodo as oauth_zenodo
+import hermes.commands.init.github_secrets as github_secrets
+import hermes.commands.init.github_permissions as github_permissions
+import hermes.commands.init.slim_click as sc
+
+
+class HermesInitFolderInfo:
+    def __init__(self):
+        self.absolute_path: str = ""
+        self.has_git: bool = False
+        self.git_remote_url: str = ""
+        self.uses_github: bool = False
+        self.uses_gitlab: bool = False
+        self.has_hermes_toml: bool = False
+        self.has_gitignore: bool = False
+        self.has_citation_cff = False
+        self.current_branch: str = ""
+
+
+def scout_current_folder() -> HermesInitFolderInfo:
+    info = HermesInitFolderInfo()
+    current_dir = os.getcwd()
+    info.absolute_path = str(current_dir)
+    info.has_git = os.path.isdir(os.path.join(current_dir, ".git"))
```

Comment:
Please also check if the Git command is actually available and fail gracefully - otherwise you'll run into trouble tryint to execute it.